lib/webidl2.js

```diff
@@ -195,9 +195,13 @@
 
     const EMPTY_IDLTYPE = Object.freeze({
       generic: null,
-      nullable: false,
+      nullable: null,
```

Comment:
Can you help me understand why we don’t default to false? Null seems a little odd here.